src/System.Globalization.Calendars/tests/Misc/Calendars.netstandard1.7.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Globalization;
+using System.Collections.Generic;
+
+namespace System.Globalization.Tests
+{
+
+    public static class CalendarsMoreTests
+    {
+        public static IEnumerable<object[]> Calendars_TestData()
+        {
+            //                              Calendar               yearHasLeapMonth        CalendarAlgorithmType 
+            yield return new object[] { new ChineseLunisolarCalendar()  , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new GregorianCalendar()         , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new HebrewCalendar()            , 5345  , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new HijriCalendar()             , 0     , CalendarAlgorithmType.LunarCalendar       };
+            yield return new object[] { new JapaneseCalendar()          , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new JapaneseLunisolarCalendar() , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new JulianCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new KoreanCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new KoreanLunisolarCalendar()   , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new PersianCalendar()           , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new TaiwanCalendar()            , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new TaiwanLunisolarCalendar()   , 0     , CalendarAlgorithmType.LunisolarCalendar   };
+            yield return new object[] { new ThaiBuddhistCalendar()      , 0     , CalendarAlgorithmType.SolarCalendar       };
+            yield return new object[] { new UmAlQuraCalendar()          , 0     , CalendarAlgorithmType.LunarCalendar       };
+        }
+
+        [Theory]
+        [MemberData(nameof(Calendars_TestData))]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
```

Comment:
But unix will be supported?  Might be better then to use:
```C#
[ActiveIssue(issueNumberForEnablingUnix, Xunit.PlatformID.AnyUnix)]
```
for this and others like it.